ArjunaJTS/jts/tests/classes/com/hp/mwtests/ts/jts/orbspecific/local/timeout/TerminationTest.java

```diff
@@ -192,8 +190,6 @@ public void test()
                 current.commit(true);
             else
                 current.rollback();
-
-            assertFalse(commit);
```

Comment:
This line should *not* be removed:

If commit is true then we try to commit a timed out transaction. Therefore current.commit(true) will throw an exception bypassing this assertFalse(commit). If the exception is not thrown then we want the assert to fail.

On the other hand if we try to rollback a timed out transaction then then it will (quietly) succeed and assertFalse(commit) will be true (since commit is false).